pandas/io/pytables.py

```diff
@@ -454,9 +454,9 @@ class HDFStore(StringMixin):
 
     Examples
     --------
-    >>> from pandas import DataFrame
-    >>> from numpy.random import randn
-    >>> bar = DataFrame(randn(10, 4))
+    >>> import numpy as np
```

Comment:
Yeah.. I wasn't sure either.. In some places, its explicitly listed, and in others its not. So in the few places where it seemed, it would help, I explicitly listed them.. but I can take them out as well..

Reply:
We do not, you can remove all occurences of `import numpy as np` and `import pandas as pd`

Reply:
We do not, you can remove all occurences of `import numpy as np` and `import pandas as pd`

Reply:
Apart from that, looking really good!

Reply:
Ok.. sounds good. I will remove the `import numpy as np` and `import pandas as pd`. If there is a `pd.DataFrame` or `np.random` being defined or something similar.. should I convert it back to just `DataFrame` and `random` respectively? Wondering if there is a good reference.. so I can convert these to the correct format in one shot..

Reply:
no, we always assume these as imports